Point script query docs at runtime fields (#71291)
elastic/elasticsearch: docs/reference/query-dsl/script-query.asciidoc

<titleabbrev>Script</titleabbrev>
++++

> NOTE: <<runtime,Runtime fields>> provide a very similar feature that is quite
>       a bit more flexible. You write a script to create field values and they
>       are available everywhere - <<search-fields, `fields`>>,
>       <<query-dsl, all queries>>, <<search-aggregations, aggregations>>,
>       and anything else you can think of.

Reviewer: ```suggestion
NOTE: <<runtime,Runtime fields>> provide a very similar feature that is
      more flexible. You write a script to create field values and they
      are available everywhere, such as <<search-fields, `fields`>>,
      <<query-dsl, all queries>>, and <<search-aggregations, aggregations>>.

```
